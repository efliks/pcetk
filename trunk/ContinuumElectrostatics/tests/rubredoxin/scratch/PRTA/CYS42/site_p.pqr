ATOM      0  CB  CYS    42       7.961  48.238   4.719  -0.11000   1.700      0      0      0      0
ATOM      0  HB1 CYS    42       8.882  48.813   4.968   0.09000   1.000      0      0      0      0
ATOM      0  HB2 CYS    42       7.282  48.950   4.193   0.09000   1.000      0      0      0      0
ATOM      0  SG  CYS    42       8.366  46.862   3.632  -0.23000   1.800      0      0      0      0
ATOM      0  HG1 CYS    42       7.155  46.789   3.104   0.16000   0.300      0      0      0      0
