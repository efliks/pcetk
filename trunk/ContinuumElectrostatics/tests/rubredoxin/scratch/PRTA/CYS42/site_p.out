Starting /home/mikolaj/local/bin//my_3diel_solver for molecule named scratch/PRTA/CYS42//site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (13.319, 37.4012, 4.51304)) 121 2
Finer level:    (7.92924, 47.9304, 4.12304) 101 1
Finer level:    (7.92924, 47.9304, 4.12304) 101 0.5
Finest level:   (7.92924, 47.9304, 4.12304) 101 0.25


Self energy of scratch/PRTA/CYS42//site_p in three dielectric environment = 25.9002
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/CYS42//site_p with background charge set in three dielectric environment = -1.35982
(probably in kcal/mole)



Interaction energies of scratch/PRTA/CYS42//site_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.00380117
0   1 : -9.45308e-05
1   0 : 0.135678
1   1 : 1.3545
2   0 : -0.0109418
2   1 : -0.00119475
3   0 : 0.13147
3   1 : 0.987975
4   0 : 0.0157518
4   1 : 0.0900654
5   0 : 0.000857292
5   1 : 0.0339391
6   0 : -0.000922371
6   1 : 0.00752451
7   0 : -0.00212905
7   1 : 0.00816459
8   0 : -0.000243026
8   1 : 0.00207123
9   0 : -0.00133321
9   1 : 0.0301512
10   0 : -0.000493182
10   1 : 0.00453063
11   0 : -0.00094081
11   1 : -2.50488e-05
12   0 : 5.60643e-06
12   1 : 0.00405232
13   0 : -4.12549e-06
13   1 : 0.00156876
14   0 : 2.65325e-05
14   1 : 0.00179114
15   0 : 0.000149918
15   1 : 0.00396352
16   0 : -0.235663
16   1 : 1.60064
17   0 : 51.8254
17   1 : 92.9292
18   0 : -0.0149207
18   1 : -0.000812971
19   0 : -0.0025753
19   1 : 0.01374
20   0 : 8.35311e-05
20   1 : 0.00671994
21   0 : -0.00249673
21   1 : -0.000124138
22   0 : -0.000154023
22   1 : 0.00154471
23   0 : -3.02539e-05
23   1 : 0.000899049
Total runtime: 15.3 s user 0.13 s system 
